warehouse/internal/repo/staging_test.go

```diff
@@ -522,8 +523,10 @@ func TestStagingFileIDs(t *testing.T) {
 			ID: 3,
 		},
 	}
-
-	ids := repo.StagingFileIDs(sfs)
+	var ids []int64
+	lo.ForEach(sfs, func(sf *model.StagingFile, _ int) {
+		ids = append(ids, sf.ID)
+	})
```

Comment:
I find the usage of `lo.ForEach` hindering readability in this case. 

We can always use a for loop. 
Or use `lo.Map`.

I personally prefer `repo.StagingFileIDs` since it is a common/repeated behaviour in our code.

Reply:
Yeah I was thinking to change this to the older code too